src/libs/fileDownload/FileUtils.js

```diff
@@ -152,7 +152,8 @@ const readFileAsync = (path, fileName) =>
                 return res.blob();
             })
             .then((blob) => {
-                const file = new File([blob], cleanFileName(fileName));
+                const cleanName = cleanFileName(fileName);
+                const file = new File([blob], cleanName);
```

Comment:
Why is this change necessary?

Reply:
I think this was a change I had forgotten to revert